integration_tests/integration/assess-recall.spec.js

```diff
@@ -35,6 +35,10 @@ context('Assess a recall', () => {
         fileName: 'recall-request.eml',
       },
     ],
+    missingDocuments: {
```

Comment:
Could just add this to the default get-recall.json response?

Reply:
I didnt want it to apply everywhere, as didnt want the knock on effect everything.. if you'd rather it there then sure.